proposals/common-definitions/tooling/tds.js

```diff
@@ -163,9 +163,9 @@ const validCompactTDs = [
         "type": "string",
         "forms": [
           {
-            "connection": "conn1",
-            "form": "form1",
-            "security": "sec1",
+            "connection": ["conn1"],
+            "form": ["form1"],
+            "security": ["sec1"],
```

Comment:
referencing in the forms with array is a bit weird since you cannot have OR relationship here anymore, it happens in the array just above (forms). We can say that these terms MUST be string in the form level? Not sure since we start changing the type based on the level etc.